docs/utilities/elevationUtilities.md

```diff
@@ -5,12 +5,14 @@ This is a general utilities class that allows checks to get elevation data. The
 
 #### Configuration
 
+The `elevation.prefix` is used to help filter files on Spark (all extra data files are put in the _same_ directory).
+
 ```json
 {
     "ElevationUtilities": {
         "elevation.srtm_extent": 1.0 (degree),
         "elevation.srtm_ext": "hgt" (file extension),
-        "elevation.path": "elevation"
+        "elevation.prefix": "elevation-"
```

Comment:
Are these changes associated with GenericTagCheck addition?

Reply:
No. I think it is a holdover from when I was fiddling with methods for transferring files between spark nodes.